DesignRules.md

```diff
@@ -1,4 +1,4 @@
-# The Design Rules
+# The core set of Design Rules
```

Comment:
Is core "set" niet verwarrend, is dit ook niet een "module"?

Reply:
omdat alle modules rules bevatten is in de modulaire opzet gekozen voor 1 module met een core set van design rules. deze /`core`/ is ook in de naamgeving van de rules nu meegenomen. Ik pas m dan ook nu niet aan in het kader van de TLS module review. Als er veel feedback op komt vanuit de consultatie kijken we nogmaals naar de naamgeving.